integrations/ollama/example/example.py

```diff
@@ -0,0 +1,43 @@
+from haystack import Document, Pipeline
```

Comment:
I would add some comments before starting to make this example self-contained. Something like this:

To run this example, you need to have an active instance of Ollama, with the LLM used
It is possible to do this via docker:
```
docker run -d -p 11434:11434 --name ollama ollama/ollama:latest
docker exec ollama ollama pull orca-mini
```

Reply:
Done